docs/source/mypy_daemon.rst

```diff
@@ -103,5 +99,3 @@ Limitations
   limitation. This can be defined
   through the command line or through a
   :ref:`configuration file <config-file>`.
-
-* Windows is not supported.
```

Comment:
I think that timeouts don't work on windows, so we should document that.

And file a follow-up bug. I think it is doable, but will require using overlapped IO and WaitForSingleObject. So probably something of a pain. No need to worry about it now.